FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -354,7 +377,7 @@ uint8_t ucMinAgeFound = 0U;
 		{
 			/* Does this row in the ARP cache table hold an entry for the MAC
 			address being searched? */
-			if( memcmp( pxMACAddress->ucBytes, xARPCache[ x ].xMACAddress.ucBytes, sizeof( MACAddress_t ) ) == 0 )
+			if( ( void ) memcmp( pxMACAddress->ucBytes, xARPCache[ x ].xMACAddress.ucBytes, sizeof( MACAddress_t ) ) == 0 )
```

Comment:
Why are you casting the result of `memcmp()` away when you are using it in a comparison at the end of the statement (` == 0`)?  Delete the cast to `( void )`.

Reply:
+1. Casting breaks the logic, please fix.